src/apps/queues/views.py

```diff
@@ -0,0 +1,76 @@
+# from braces.views import LoginRequiredMixin
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.core.exceptions import PermissionDenied, ValidationError
+# from django.core.urlresolvers import reverse
+from django.db.models import Q
+from django.urls import reverse
+from django.forms.utils import ErrorList
+from django.http import HttpResponse
+from django.http import HttpResponseRedirect
+from django.views.generic import ListView, CreateView, UpdateView, DeleteView
+from pyrabbit.http import HTTPError
+
+from .forms import QueueForm
+from .models import Queue
+from . import rabbit
+
+
+class QueueFormMixin(object):
+    def get_success_url(self):
```

Comment:
why do we need this function? why can't we just use `reverse` each time?

Reply:
It looks like `get_success_url` looks for either `self.success_url` or `self.object.get_absolute_url()`
so instead of overwriting this function, we could just set the variable? like:
```python
class QueueFormMixin:
    success_url = reverse('queues:list')
```
What do you think about that instead? or do we wanna override the function? I could kinda go either way on it, It just feels weird to me that we are inheriting from `object` and overwriting the functions of other classes... Maybe I'm just biased against mixins....